dev-resources/logback.xml

```diff
@@ -5,11 +5,11 @@
         </encoder>
     </appender>
 
-    <logger name="puppetlabs" level="error"/>
+    <logger name="puppetlabs" level="trace"/>
```

Comment:
yeah, that might be the solution here.  @justinstoller if you've observed that this does affect logging tests then I was probably just wrong.